src/test/kotlin/com/terraformation/backend/accelerator/SubmissionNotifierTest.kt

```diff
@@ -108,10 +109,12 @@ class SubmissionNotifierTest : DatabaseTest(), RunsAsUser {
       insertSubmissionDocument()
       val documentId = insertSubmissionDocument()
 
+      val deliverable = mockDeliverable()
```

Comment:
Is this the correct way to get the deliverable @sgrimm @nickgraz? I'm not sure how to do it here

Reply:
Since the deliverable is added to the test DB on line 87, you can either pull it from the database ([like this](https://github.com/terraware/terraware-server/blob/main/src/test/kotlin/com/terraformation/backend/accelerator/SpeciesNotifierTest.kt#L102)) or simply create an "expected deliverable" where you need it.